ReactAndroid/src/main/java/com/facebook/react/modules/toast/ToastModule.java

```diff
@@ -25,6 +27,8 @@
  */
 public class ToastModule extends ReactContextBaseJavaModule {
 
+  private Map<Integer, Toast> mToasts = new HashMap<>();
```

Comment:
If I show many toasts and never call cancel, this map will grow indefinitely, holding references to the Android Toast classes which might consume quite a bit of memory. Is that right?